spec/views/organisations/form.html.erb_spec.rb

```diff
@@ -3,7 +3,8 @@
 describe "organisations/_form.html.erb", :type => :view do
   before :each do
     @categories_start_with = {what_they_do: 'alligator', who_they_help: 'crocodile', how_they_help: 'iguana'} 
-    @organisation = Organisation.new(name: 'Sarah Plain and Tall Foundation')
+    @organisation = Organisation.new(name: 'Sarah Plain and Tall Foundation', 
+                                     description: 'A very charitable organisation')
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)